tests/disruption_helpers.py

```diff
@@ -71,6 +74,14 @@ def delete_resource(self, resource_id=0):
             resource_count=self.resource_count, timeout=300
         )
 
+    def delete_app_resource(self, resource_id=0):
```

Comment:
I try to use "delete_resource()" method.
However, this function can delete components only from openshift-storage name space. 
Because this line:
POD = ocp.OCP(kind=constants.POD, namespace=config.ENV_DATA['cluster_namespace'])

Reply:
If you are looking to delete the app pod, there is one more way to do it. 
We can get the app pod object and delete the pod.
new_pod_obj = get_pod_obj(name=<pod_name>, namespace=<namespace>)
new_pod_obj.delete()